web/cypress/e2e/app.cy.ts

```diff
@@ -40,15 +40,19 @@ describe("ARC Portal UI authenticated", () => {
 });
 
 describe("Setting chosen name for user", () => {
+  beforeEach(() => {
+    cy.loginAsAdmin();
+    cy.clearChosenName();
+    cy.visit("/");
+  });
```

Comment:
Doing it this way is more 'cypressy' as it removes the need to make a new custom function